MekHQ/src/mekhq/Utilities.java

```diff
@@ -1154,7 +1154,7 @@ public static void unscrambleEquipmentNumbers(Unit unit) {
                     for(int equipNum : equipNums) {
                         i++;
                         Mounted m = unit.getEntity().getEquipment(equipNum);
-                        if (!allMunitions && (part instanceof AmmoBin)
```

Comment:
On this branch, we are checking for MissingParts, so it can't be an AmmoBin.

Reply:
Should be MissingAmmoBin.

Reply:
@sixlettervariables allMunitions is always false when we get to that point of the code. That looks like a bug no? We set it to false in line 1119 and then of that for loop we never change the value again. Any idea what's the correct logic? (I can't figure out).